src/cli/nexus-prisma.ts

```diff
@@ -66,7 +66,7 @@ generatorHandler({
           solution: `Create a configuration file in one of the following locations:\n\n${renderList(supportedSettingsModulePaths)}\n\nThen add the following code:\n\n${renderCodeBlock(dindist`
             import { settings } from 'nexus-prisma/generator'
 
-            settings.change({
```

Comment:
Is this a straight up bug? How was this working before?

Reply:
I started seeing that this Warning has an issue with this bug report #196
You already have the right one in the docs https://nexus.prisma.io/settings-gentime#usage
Also a wrong one on the same page https://nexus.prisma.io/settings-gentime#prisma-schema-file-generator-config

And you already call the change here https://github.com/prisma/nexus-prisma/blob/main/src/generator/Settings/Gentime/singleton.ts#L23

Reply:
Thanks👌 

Please make this its own PR. And we should add a test that catches this bug 🙈.